src/docker/workflows/cp-schema-registry.test.ts

```diff
@@ -85,6 +86,10 @@ describe("docker/workflows/cp-schema-registry.ts ConfluentPlatformSchemaRegistry
 
     workflow = ConfluentPlatformSchemaRegistryWorkflow.getInstance();
 
+    // Set up the Kafka workflow to have the correct imageTag for tests
+    const kafkaWorkflow = LocalResourceWorkflow.getKafkaWorkflow();
+    kafkaWorkflow.imageTag = LOCAL_KAFKA_IMAGE_TAG.defaultValue;
```

Comment:
Let's remove this in favor of setting the `imageTag` property for the Kafka and SR workflow classes, which causes all of the tests to pass as expected for me locally:
- `imageTag = LOCAL_KAFKA_IMAGE_TAG.value;` under here: https://github.com/confluentinc/vscode/blob/afe7471dcb3ff1c0acc2792a0412796d5008ef4e/src/docker/workflows/confluent-local.ts#L31-L33
- `imageTag = LOCAL_SCHEMA_REGISTRY_IMAGE_TAG.value;` under here: https://github.com/confluentinc/vscode/blob/afe7471dcb3ff1c0acc2792a0412796d5008ef4e/src/docker/workflows/cp-schema-registry.ts#L37-L39

I think we'll still need to keep the `this.imageTag = <setting>.value` lines in each of those workflows during the `start()` and `stop()` methods to account for changing settings after the workflow instances have been created.